R/bayesfactor_parameters.R

```diff
@@ -185,7 +185,12 @@
 #' @author Mattan S. Ben-Shachar
 #'
 #' @export
-bayesfactor_parameters <- function(posterior, ...) {
+bayesfactor_parameters <- function(posterior,
+                                   prior = NULL,
+                                   direction = "two-sided",
+                                   null = 0,
+                                   ...,
+                                   verbose = TRUE) {
```

Comment:
I would use as few arguments as necessary for the generic. Else, _all_ other methods _at least_ have to include all these

Reply:
Yes, these are the minimally required inputs and argument 😀